.storybook/snapshots/test-runner.ts

```diff
@@ -0,0 +1,37 @@
+import { Page } from 'playwright-core';
+import { TestRunnerConfig, TestContext } from '@storybook/test-runner';
+import { toMatchImageSnapshot } from 'jest-image-snapshot';
+
+const customSnapshotsDir = `${process.cwd()}/.storybook/snapshots/__snapshots__`;
+
+/**
+ * See https://storybook.js.org/docs/react/writing-tests/test-runner#test-hook-api-experimental
+ * to learn more about the test-runner hooks API.
+ */
+const renderFunctions: TestRunnerConfig = {
+  setup() {
+    expect.extend({ toMatchImageSnapshot });
+  },
+  async postVisit(page: Page, context: TestContext) {
+    if (context.id === 'locationbias--loading') {
+      return;
+    }
+    
+    const isMapboxMapStory = context.id.startsWith('mapboxmap--');
+    if (isMapboxMapStory) {
+      await page.waitForTimeout(7500);
+    }
+
+    const image = await page.screenshot();
+    expect(image).toMatchImageSnapshot({
+      customSnapshotsDir,
+      customSnapshotIdentifier: context.id,
+      failureThreshold: context.id === 'geolocation--loading' || isMapboxMapStory
+        ? 0.005
```

Comment:
out of curiosity, does this mean the difference threshold between two snapshots?

Reply:
exactly, this means that if the image that's currently taken is more than 0.005% pixels different from the image saved in the repo, then it'll fail and trigger a snapshot update